app/src/test/java/nextstep/jwp/domain/TokenTest.java

```diff
@@ -0,0 +1,23 @@
+package nextstep.jwp.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class TokenTest {
+
+    @Test
+    void value() {
+        //given
+        String pureToken = "Token";
+        ManualStrategy manualStrategy = new ManualStrategy("Token");
+        Token token = Token.fromStrategy(manualStrategy);
```

Comment:
```suggestion
        final String pureToken = "Token";
        TokenStrategy tokenStrategy = new TokenStrategy() {
            @Override
            public String token() {
                return pureToken;
            }
        };
```

`ManualStrategy`의 사용처가 테스트 밖에 없더군요. 테스트를 위한 프로덕션 코드 작성보다는, 이렇게 임시로 사용할 익명 페이크 클래스를 테스트에 사용하는 것은 어떨까요?

Reply:
오..익명 클래스 하나 더 배워갑니다!